spec/src/main/asciidoc/jakarta-stl.adoc

```diff
@@ -1,5 +1,8925 @@
+:sectnums!:
+== Preface
+
+This is the Jakarta Server Pages™ Standard Tag
+Library 2.0 (JSTL 2.0) specification, developed by the JSR-52 expert
+group under the Java Community Process. +
+See http://www.jcp.org.
+
+== Related Documentation
+
+Implementors of JSTL and authors of JSP pages
+may find the following documents worth consulting for additional
+information:.
+
+[width="100%",cols="50%,50%",]
+|===
+|Jakarta Server Pages (JSP)
+|http://java.sun.com/jsp
+
+|Jakarta Servlet Technology
+|http://java.sun.com/servlet
+
+|Java 2 Platform, Standard Edition
+|http://java.sun.com/j2se
+
+|Java 2 Platform, Enterprise Edition
+|http://java.sun.com/j2ee
+
+|JavaBeans
+|http://java.sun.com/beans
+
+|JDBC
+|http://java.sun.com/jdbc
+
+|Java Technology and XML
+|http://java.sun.com/xml
+
+|XPath specification
+|http://www.w3.org/TR/xpath
+
+|XML home page at W3C
+|http://www.w3.org/XML
+
+|HTML home page at W3C
+|http://www.w3.org/MarkUp
+
+|XML.org home page
+|http://www.xml.org
+|===
+
+== Typographical Conventions
+
+
+
+[width="100%",cols="50%,50%",options="header",]
+|===
+|Font Style |Uses
+|Italic |Emphasis,
+definition of term.
+
+| _Monospace_ |Syntax,
+code examples, attribute names, Java language types, API, enumerated
+attribute values.
+|===
+
+== Acknowledgments
+
+The JavaServer Pages™ Standard Tag Library
+(JSTL) specification is the result of collaborative work involving many
+individuals, all driven by a common goal of designing the best libraries
+possible for the JSP page author community.
```

Comment:
Remove JSP term

Reply:
Changed to: `possible for the Jakarta Server Pages author community`